### PR TITLE
Add milvus

### PR DESCRIPTION
**Pull request by Mridulbirla13**
Thanks for contributing to the [Hacktoberfest Swag List](https://hacktoberfestswaglist.com/) :smiley: :tada:! Before submitting your pull request, please check off as many of the items below as you can:

1. [x] I have read the [Contributing.md](../CONTRIBUTING.md) file and formatted this PR correctly
2. [x] I'm not adding a company from the blocklist
3. [x] I make sure to fix things promptly if an error or suggestion comes up

Thanks and Happy Hacktoberfest! :tada:
Tagging @crweiner to take a look. :eyes:

### CODE REVIEW

**Comment by crweiner** [comment]
Hey @Mridulbirla13! 👋 Thanks for your PR to the Hacktoberfest Swag List! Would you be able to tackle the [md flags from Code Climate](https://codeclimate.com/github/crweiner/hacktoberfest-swag-list/pull/405)? I setup that robotic overlord to make sure that the markdown -> HTML gets rendered properly and consistently for the site. 🤖 

Thank you!

**Comment by Mridulbirla13** [comment]
Hey @crweiner , I have solved issues stated by Code Climate Bot.  Can you further guide me, how to further resolve conflict?

File: README.md [hunk 1]
```diff
@@ -346,6 +368,17 @@ See [**Contributing.md**](./CONTRIBUTING.md) to see how to format your pull requ
...
+
+- **Swag**: a sticker pack, a digital Milvus contributor badge, and a Milvus T-shirt
+- **Requirements**: Sign up for the event on the [Hacktoberfest website](https://milvus.io/hacktoberfest-2021/)  and just start contributing to any open source project - it’s as simple as that! If you’re new to the open source world, no worries, we’ve got you covered with [Guide](https://discuss.milvus.io/t/join-hacktoberfest-2021-with-us/72).
+  - By contributing to the Milvus project between October 1 and October 31 you’ll also earn some additional swag and prizes sponsored by Zilliz
+- **How to sign up**:
+  - Sign up [here](https://milvus.io/hacktoberfest-2021/)
+  - Join Discussions by creating [Milvus Forum Account](https://discuss.milvus.io/c/hacktoberfest/9) and interact with community members.
+- **Issues**: Issues are stated at Issues Section of [Hacktoberfest website](https://milvus.io/hacktoberfest-2021/).
+- **Notes**: Check out [Milvus Community Forum](https://discuss.milvus.io/c/hacktoberfest/9)
+
```
Comment (crweiner) [inline README.md:381]: Following the [Contributing guidelines](https://hacktoberfestswaglist.com/contributing), I'd love to see you format this entry again. Use the examples from line 280-287 above.

File: README.md [hunk 2]
```diff
@@ -256,13 +277,14 @@ See [**Contributing.md**](./CONTRIBUTING.md) to see how to format your pull requ
...
-- **How to sign up**: Complete your PR after accepted please fill out this [form](https://forms.gle/DVkrWCu4yD2wMPmE8). All PRs that are contributing value will get an Earthly sticker!
-- **Issues**: [Grab an issue](https://github.com/earthly/earthly/issues?q=is%3Aissue+is%3Aopen+label%3Ahacktoberfest)
-- **Notes**: If you need any assistance, [join their Slack channel](https://earthly.dev/slack). See their [Blog post](https://earthly.dev/blog/hacktoberfest-2021/) for more details.
+- **Requirements**: Contribute to at least one merged pull request.
+- **Swag**: a sticker pack & a digital Milvus contributor badge.
+- **How to sign up**: Sign up and get more info [here](https://milvus.io/hacktoberfest-2021/)
+  - Join Discussions by creating a [Milvus Forum Account](https://discuss.milvus.io/c/hacktoberfest/9) and interact with community members.
+- **Issues**: Issues tagged for Hacktoberfest are available on the [Milvus Hacktoberfest website](https://milvus.io/hacktoberfest-2021/).
+- **Notes**: Check out [Milvus Community Forum](https://discuss.milvus.io/c/hacktoberfest/9) and view the [Hacktoberfest thread here.](https://discuss.milvus.io/t/join-hacktoberfest-2021-with-us/72)
```
Comment (crweiner) [inline README.md:287]: I formatted this to be more in line with the [Contributing guidelines](https://hacktoberfestswaglist.com/contributing). Using this as a template, please edit the other entries below.

File: README.md [hunk 3]
```diff
@@ -387,6 +420,20 @@ See [**Contributing.md**](./CONTRIBUTING.md) to see how to format your pull requ
...
+  - By contributing to the Milvus project between October 1 and October 31 you’ll also earn some additional swag and prizes sponsored by Zilliz
+  - Swag tiers
+    - **For Top Contributor (Best Quality PR)** : A Logitech Pro X Keyboard and a physical Hacktoberfest badge.
+    - **For Top Contributor (Most Difficult PR) :trophy:** : A DJI Drone if you complete at least 1 PR merged with the issue tagged #advanced .
+- **How to sign up**:
+  - Sign up [here](https://milvus.io/hacktoberfest-2021/)
+  - Join Discussions by creating [Milvus Forum Account](https://discuss.milvus.io/c/hacktoberfest/9) and interact with community members.
+- **Issues**: Issues are stated at Issues Section of [Hacktoberfest website](https://milvus.io/hacktoberfest-2021/).
+- **Notes**: Check out [Milvus Community Forum](https://discuss.milvus.io/c/hacktoberfest/9)
+
```
Comment (crweiner) [inline README.md:436]: Following the [Contributing guidelines](https://hacktoberfestswaglist.com/contributing), I'd love to see you format this entry again. Use the examples from line 280-287 above.

**Review by crweiner — CHANGES_REQUESTED** [review]
Hey @Mridulbirla13! There's a few things I'd like you to update, please. Please let me know when you're ready for me to take another look! 👀 Thanks! 👍

**Comment by crweiner** [comment]
I appreciate your work on this! Milvus was just added in PR #408. I highly encourage you to look for more companies offering unique Hacktoberfest swag, because there's lots more out there! 🔍 

Thank you again and I hope you have a marvelous Hacktoberfest!!

**Pull request closed without merging**